habito/habito.py

```diff
@@ -140,6 +140,41 @@ def add(name, quantum, units):
                       created_date=datetime.now(), quantum=quantum,
                       units=units, magica="")
 
+@cli.command()
+@click.argument("id", type=click.INT)
+@click.option('--name', '-n', help="The new name (leave empty to leave unchanged)")
+@click.option('--quantum', '-q', help="The new quantum (leave empty to leave unchanged)", type=click.FLOAT)
+def edit(id, name, quantum):
+    try:
+        habit = HabitModel.get(HabitModel.id == id)
+    except DoesNotExist:
+        click.echo("The habit you're trying to edit does not exist!")
+        raise SystemExit(1)
+    habit.name = name.strip() or habit.name
+    habit.quantum = quantum or habit.quantum
+    habit.save()
+    click.echo("Habit with id {} has been saved with name: {} and quantum: {}".format(id, habit.name, habit.quantum))
+
+
+@cli.command()
+@click.argument("id", type=click.INT)
+@click.option("--keeplogs", is_flag=True, default=False)
+def delete(id, keeplogs):
```

Comment:
Makes sense. We already have the `active` field, we should merge the `keeplogs` and archive scenarios; they seem similar in purpose?

I agree with you, privacy is not in the immediate scope of this tool.